playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -102,12 +112,26 @@
       state: link
     when: ca_crt_stat.stat.isreg and not ca_bundle_stat.stat.exists
 
+  # Run the upgrade hook prior to restarting services/system if defined:
+  - debug: msg="Running upgrade hook {{ openshift_master_upgrade_hook }}"
```

Comment:
I almost wonder if we need to still make this more descriptive. If for no other reason it would make it less confusing if we need to add additional hooks at other parts of the master upgrade process.